<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1049</width>
    <height>877</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>logo.ico</normaloff>logo.ico</iconset>
  </property>
  <widget class="QLabel" name="logistics_label_orderID">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Sans Serif</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
   <property name="text">
    <string>订单编号:</string>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
  </widget>
  <widget class="QTextEdit" name="logistics_textEdit_orderID">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>60</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="logistics_label_demo">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>421</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Sans Serif</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
   <property name="text">
    <string>Mode</string>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
  </widget>
  <widget class="QLabel" name="logistics_label">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>10</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Sans Serif</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="whatsThis">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;New Order&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
   <property name="text">
    <string>新物流</string>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
  </widget>
  <widget class="QLabel" name="logistics_label_vendorID">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Sans Serif</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
   <property name="text">
    <string>物流商编号:</string>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
  </widget>
  <widget class="QTextEdit" name="logistics_textEdit_vendorID">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>110</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="logistics_tableWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>310</y>
     <width>881</width>
     <height>561</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Sans Serif</family>
     <pointsize>11</pointsize>
     <kerning>false</kerning>
    </font>
   </property>
   <property name="cursor" stdset="0">
    <cursorShape>IBeamCursor</cursorShape>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhLatinOnly</set>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::AllEditTriggers</set>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
   <property name="rowCount">
    <number>30</number>
   </property>
   <property name="columnCount">
    <number>3</number>
   </property>
   <attribute name="horizontalHeaderCascadingSectionResizes">
    <bool>true</bool>
   </attribute>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>250</number>
   </attribute>
   <attribute name="horizontalHeaderMinimumSectionSize">
    <number>250</number>
   </attribute>
   <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
    <bool>true</bool>
   </attribute>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderCascadingSectionResizes">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderDefaultSectionSize">
    <number>30</number>
   </attribute>
   <attribute name="verticalHeaderMinimumSectionSize">
    <number>30</number>
   </attribute>
   <attribute name="verticalHeaderStretchLastSection">
    <bool>false</bool>
   </attribute>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <column/>
   <column/>
   <column/>
  </widget>
  <widget class="QPushButton" name="logistics_pushButton_autoFill">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>310</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Sans Serif</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>自动带入</string>
   </property>
  </widget>
  <widget class="QPushButton" name="logistics_pushButton_generateCSV">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>360</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Sans Serif</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>导出数据</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="logistics_textEdit_trackingNumber">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>160</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="logistics_label_trackingNumber">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Sans Serif</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
   <property name="text">
    <string>包裹追踪码:</string>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
  </widget>
  <widget class="QLabel" name="logistics_label_trackingURL">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>160</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Sans Serif</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
   <property name="text">
    <string>包裹追踪网址:</string>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
  </widget>
  <widget class="QTextEdit" name="logistics_textEdit_trackingURL">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>160</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="logistics_label_destination">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>60</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Sans Serif</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
   <property name="text">
    <string>Destination:</string>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
  </widget>
  <widget class="QLabel" name="logistics_label_postedBy">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Sans Serif</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
   <property name="text">
    <string>寄件人:</string>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
  </widget>
  <widget class="QTextEdit" name="logistics_textEdit_postedBy">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>210</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="logistics_label_vendorName">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>110</y>
     <width>301</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Sans Serif</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="logistics_pushButton_updateSystem">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>410</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Sans Serif</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>更新系统</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="logistics_checkBox_air">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>60</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Sans Serif</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>空</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="logistics_checkBox_surface">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>60</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Sans Serif</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>陆/海</string>
   </property>
  </widget>
  <widget class="QLabel" name="logistics_label_postedAt">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>210</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Sans Serif</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
   <property name="text">
    <string>寄件日期:</string>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
  </widget>
  <widget class="QDateEdit" name="logistics_dateEdit_postedAt">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>210</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Sans Serif</family>
     <pointsize>11</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="logistics_label_status">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Sans Serif</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
   <property name="text">
    <string>包裹状态:</string>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
  </widget>
  <widget class="QLabel" name="logistics_label_ETA">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>210</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Sans Serif</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
   <property name="text">
    <string>抵达:</string>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
  </widget>
  <widget class="QDateEdit" name="logistics_dateEdit_ETA">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>210</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Sans Serif</family>
     <pointsize>11</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QCheckBox" name="logistics_checkBox_posted">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>260</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Sans Serif</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>已寄出</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="logistics_checkBox_exportCustom">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>260</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Sans Serif</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>出关中</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="logistics_checkBox_international">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>260</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Sans Serif</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>国际运输中</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="logistics_checkBox_importCustom">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>260</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Sans Serif</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>入关中</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="logistics_checkBox_delivering">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>260</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Sans Serif</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>目的地配送中</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
